.github/workflows/gradle-build-pr.yml

```
@@ -27,7 +31,7 @@ jobs:
       - name: "Checkout code"
         uses: actions/checkout@v2
       - name: "Run Connected test via Gradle"
-        uses: broadinstitute/datarepo-actions@0.14.0
+        uses: broadinstitute/datarepo-actions@sh-DR-1114-check-before-auth
```

Change all these to what the new tag will be after the action is merged. 
```suggestion
        uses: broadinstitute/datarepo-actions@0.20.0
```